reports/summary-2020.md

```diff
@@ -0,0 +1,814 @@
+  maturity
+- Expand on async meeting adoption and establish best practices 
+- Improving subproject communication and connective glue back to the sponsoring
+  SIG
+- Clarify conditions for archiving or putting a subproject in maintenance mode
+- Connecting with the end user community and understanding what they want to
+  hear from upstream
+  
+    
+### Refining the Contributor Lifecycle
+
+There are a number of ways to define membership to the Kubernetes project
+and its community groups. The project's community groups must define
+their own membership terms and then various other activities and roles.
+Membership can drive:
+
+- Kubernetes GitHub Organization membership 
+- Consensus 
+- Voting
+
+**Project Membership**
+
+The Kubernetes project defines [membership] as contributors added to to one of 
+the four
+GitHub organizations. Membership grants a contributor access to test their pull
+requests, among other benefits. Membership also defines
+a member's role within the project based on their place on the [Contributor
+Ladder][ldr]. Membership does not grant voting rights for Steering Committe and
+governance matters; there is a separate set of criteria for [elections].
+
+There are two areas to address: early membership (onboarding) and sustainable 
+membership (ongoing contributor activity).
+
+**Early Membership**
+
+When new contributors onboard to the project, they often ask how to become a 
+member of a SIG. Community group membership is currently ambiguous, and the 
+options available are overwhelming for new contributors. Many SIGs reported that
+"joining the SIG mailing list" is how they define membership.
+Other responses included Slack channel membership and OWNERs files. Some groups
+requested council from the Steering Committee on defining membership.
+
+Most community groups do not have a voting process as they are driven by 
+consensus from OWNERs files and the community-at-large. However, in some cases, 
+particularly leadership transitions, community groups voted to decide.
+
+**Sustainable Membership**
+
+While onboarding new contributors is a critical part of a sustainable open
+source project, contributors shift focus, change jobs, or
+step away from the project for a variety of other reasons. In late 2019, the 
+Steering Committee updated our guidelines to include an [emeritus status]. 
+Emeritus community members have stepped away from the project, but are still 
+recognized for their work. Giving people the ability to step away gracefully 
+helps ensure an overall healthy and active community.
+
+In 2020, 185 members stepped down or were removed for inactivity (18
+months with 0 activity across any GitHub organization). The Kubernetes project 
+has around 1300 active members at any given time.   
+
+Joining the project's GitHub organizations is celebrated, but
+there are no regular means of celebration or recognition for those that join a
+SIG, community group, or [achievement for climbing the ladder].
```

Comment:
This sounds more like a "point for further research" rather than something we've done?